community/kernel/src/main/java/org/neo4j/kernel/impl/store/kvstore/RotationState.java

```diff
@@ -49,6 +50,9 @@ String stateName()
 
     static final class Rotation<Key> extends RotationState<Key>
     {
+        // visible for testing
+        static long MAX_ROTATION_DURATION_MILLIS = 1000 * 10 * 10; // 10 minutes
```

Comment:
Why not make this an `@Internal` configuration? There might come a day, where a field agent will want to change this.